install_script.sh.template

```diff
@@ -777,22 +738,6 @@ if [ -n "$DD_AGENT_FLAVOR" ]; then
     agent_flavor=$DD_AGENT_FLAVOR #Eg: datadog-iot-agent
 fi
 
-datadog_installer=
-if [ -n "$DD_INSTALLER" ]; then
```

Comment:
I'm not sure how this is supposed to work with the refactors, but this is likely why we needed this change:  https://github.com/DataDog/auto_inject/pull/774